src/components/GlobalSignIn/blocks/Login.tsx

```diff
@@ -0,0 +1,49 @@
+import { faSignIn } from "@fortawesome/free-solid-svg-icons";
+import { P1, Button, AnchorButton } from "@deskpro/deskpro-ui";
+import { nbsp } from "../../../constants";
+import { Invalid } from "../../common";
+import type { FC } from "react";
+import type { AnyIcon } from "@deskpro/deskpro-ui";
+import type { Maybe } from "../../../types";
+
+export type Props = {
+  url: string;
+  error: Maybe<string>;
```

Comment:
To reduce complexity, I think we should be more explicit in what we expect, this is either a `string|undefined` anything else just makes it harder to understand what's going on. That may cause typing issues up the chain, but that's fine and should be addressed there.

Reply:
But in this case, I don't think that the complexity is high. But in general, I also try to keep it more simple

Reply:
Not sure I agree. For someone new to look at this code, they're not going to know what `Maybe` is, so they'd have to follow that chain. `Maybe` is `T|undefined|null` which is one more type than to juggle than just `string|undefined` so whilst `null` and `undefined` are similar, they are different which is one more case that the code may have to account for.
It's all minor, but it all adds up.

Reply:
Don't be hard on me))) We don't often have new developers. You look once and you already know

Reply:
That's not my intention. But 3 types is harder to manage than 2 types. So we might as well keep it simple and set it to what we want it to be which I guess is `string|undefined` or `string|null` but not both.